src/renderer/Player/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { useEngine } from '../../engine';
+import { PlayerMoveOptions } from '../../engine/event/PlayerMove';
+import { useSelector } from '../../store/hooks';
+import { selectPlayerName, selectPlayerPosition } from '../../store/reducers/player';
+
+import './player.css';
+
+const Player: React.FC = () => {
+  const { trigger } = useEngine();
+  const name = useSelector(selectPlayerName);
+  const position = useSelector(selectPlayerPosition);
+
+  return (
+    <div id="player" style={{ left: position.left * 80, top: position.top * 80 }}>
```

Comment:
This size (`80`) needs to be a fixed constant somewhere, maybe even an env setting?